lib/tag-handler.js

```diff
@@ -28,14 +28,13 @@ module.exports.tagHandler = (command) => async ({ github, verbose, number }) =>
   try {
   // default to the top 3 commits in master
     const commits = _exec(`git log --no-merges --format='%s' -n ${number}`).toString().trim().split('\n')
-    const versionBump = commits.find((commit) => commit.startsWith('version'))
+    const [versionBump] = commits.map((commit) => commit.match(versionPattern)).filter((c) => c)
```

Comment:
🤔 what if a commit says:

> deps - upgrade @eqworks/lumen-table to 1.2.0

Reply:
it will try to create the tag but because it doesn't match with json, it won't